### DISCUSSION TITLE
Preview Mode redirects to 404 on production

### DISCUSSION THREAD

**Discussion by scdozer** [vercel/community#336] [Help]
I am having an issue where my preview mode urls with a secret are working on localhost but redirect to a 404 on prod/any vercel build. 

-I am using Contentful/Next/Vercel. trying to see draft pages from contentful doesnt work unless the page was previously published and caused a re-build. 

-My env is set in vercel correctly. 
-urls are https
-I can see the preview cookie is set
-falback is set to false on my dynamic route page

**Comment by samcx** [comment]
Apologies for the late response!

Can you confirm is this is still an issue?
